src/install.rs

```diff
@@ -254,6 +259,31 @@ fn write_firstboot_kargs(mountpoint: &Path, args: &str) -> Result<()> {
     Ok(())
 }
 
+/// Write persistent kernel arguments.
```

Comment:
One thing we need to keep in mind is this needs to trigger a rerun of `zipl` on s390x.

Hmm...I am increasingly thinking what we want is like a special systemd unit for zipl that checks the mtime of the BLS config and runs `zipl` if it's changed or something like that.

Reply:
Indeed, after installation we do write some s390x specific args parsed from `/proc/cmdline`

Reply:
Ahh yup, goot catch. Hmm, let's circle back on this once https://github.com/coreos/coreos-installer/pull/153 merges? It'll make it easier to run `zipl`. I can open an issue once this merges.

Reply:
Sorry, haven't seen this message begore. So, i've already started working on patch to remove all logic in #153 `zipl.rs` in favour of using `--append-karg`.